lib/ActivityStreams.js

```diff
@@ -411,6 +432,17 @@ ActivityStreams.prototype = {
       this._prefsProvider.actionHandler(args);
     };
     this.on(CONTENT_TO_ADDON, this._contentToAddonHandlers);
+
+    simplePrefs.on("", prefName => {
```

Comment:
hmm is it possible use the PrefsProvider to do this? that way we have a communal place where the prefs are being changed? @k88hudson thoughts?